performance/scripts/preScript.js

```diff
@@ -0,0 +1,28 @@
+const http = require('http');
+
+module.exports = async function (context) {
+
+    const hostname = 'demo';
+    const port = '8080';
+
+    return new Promise(resolve => {
+        function poll() {
+            setTimeout(() => {
+                http.get(`http://${hostname}:${port}`, (res) => {
+                    const { statusCode } = res;
+                    if (statusCode === 200) {
+                        context.log.info(`App available on ${hostname}:${port}.`);
+                        resolve();
+                    } else {
+                        poll();
+                    }
+                }).on('error', () => {
+                    context.log.error(`App not yet available on ${hostname}:${port}.`);
+                    poll();
+                });
+            }, 5000);
```

Comment:
5s is misschien relatief lang, zeker voor de eerste poging?

Reply:
npm install duurt gemiddeld 15s dus lijkt mij voldoende.

Reply:
Ahja die moet nog ne install doen op bamboo. Maar lokaal zeker niet? Lokaal is die normaal gezien vrij snel opgestart, waardoor ge dus 5s moet wachten ipv. bv. 1s